common.lic

```diff
@@ -590,6 +590,19 @@ module DRC
 
   def play_song?(settings, song_list, worn = true, skip_clean = false, climbing = false)
     instrument = worn ? settings.worn_instrument : settings.instrument
+
+    if UserVars.instrument.nil?
+      UserVars.instrument = instrument
+    end
+
+    UserVars.instrument = instrument unless UserVars.instrument == instrument
```

Comment:
I don't think this will work. Isn't this the equivalent to:

```suggestion
    UserVars.instrument = instrument if UserVars.instrument != instrument
```

Which means they'll always be set to equal if they are not already equal, i.e. I don't think the condition on line 600 can ever happen.

Reply:
Hmmmm Ok, back to the thinking cap!

Reply:
With the new block from 600, do we need this at all now?

Reply:
Check current version. it tests nice and clean. Tested by deleting the var, ran performance, the var was set, no fuss.

Tested by renaming the var, and running performance, picked up the change, set the var, picked song again.